src/commands/server/userinfo.ts

```diff
@@ -14,16 +15,43 @@ export const command: ICommand = {
             : message.member;
 
       if (!target)
-         return QuickEmbed(message, `Could not find user \`${args.join(' ')}\``);
-
-      const embed = createFooter(message)
-         .setTitle('User info')
-         .setDescription(`User info for ${target}`)
-         .setThumbnail(target.user.avatarURL({ dynamic: true, size: 4096 }))
-         .addField('User ID', `\`${target.id}\``)
-         .addField('Created at', target.user.createdAt.toUTCString())
-         .addField('Joined at', target.joinedAt.toUTCString());
+         return sendErrorEmbed(message, `Could not find user \`${args.join(' ')}\``);
 
+      const embed = await createEmbed(message, target);
       message.channel.send(embed);
    }
 };
+
+async function createEmbed(message: Message, target: GuildMember): Promise<MessageEmbed> {
+   const embed = createFooter(message)
+      .setTitle('User info')
+      .setDescription(`User info for ${target}`)
+      .setThumbnail(target.user.displayAvatarURL({ dynamic: true }))
```

Comment:
I added size:4096 to make sure we get the highest quality we can but im not sure if the api changed.